### PR TITLE
166. Displaying Recipe Items and Using Network Images

### PR DESCRIPTION
**Pull request by MapleSyyrup**
- Display photos of recipe items using network images

Sample screenshots:
image
image

### CODE REVIEW

File: lib/widgets/meal_item.dart [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  @override
+  Widget build(BuildContext context) {
+    return InkWell(
+      onTap: selectMeal,
+      child: Card(
+        shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.circular(15),
+        ),
+        elevation: 4,
+        margin: EdgeInsets.all(10),
```
Comment (Nei1eveN) [inline lib/widgets/meal_item.dart]: const

**Comment by Nei1eveN** [comment]
@MapleSyyrup can you try record a video with the simulation for these changes? I want to see how it proceeds to details page when an item is clicked.

**Comment by MapleSyyrup** [comment]
@Nei1eveN  please see attached
[166.zip](https://github.com/MapleSyyrup/Meals_App/files/5684375/166.zip)

File: lib/screens/category_meals_screen.dart [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import '../widgets/meal_item.dart';
+
+import '../dummy_data.dart';
+
+class CategoryMealsScreen extends StatelessWidget {
+  static const routeName = '/category-meals';
+  // final String categoryId;
+  // final String categoryTitle;
+
+  // CategoryMealsScreen(this.categoryId, this.categoryTitle);
```
Comment (Nei1eveN) [inline lib/screens/category_meals_screen.dart:11]: do you still need this?

Comment (MapleSyyrup) [reply lib/screens/category_meals_screen.dart:11]: I'm using it for reference for now

Comment (Nei1eveN) [reply lib/screens/category_meals_screen.dart:11]: if the succeeding pull requests make no use of those, please remove them.

**Pull request merged**